ceremonies/src/tests.rs

```diff
@@ -313,6 +310,17 @@ fn assert_error(actual: DispatchResult, expected: Error::<TestRuntime>) {
     }.unwrap(), expected.as_str());
 }
 
+fn get_meetups(cid: CommunityIdentifier) -> Vec<Vec<AccountId>> {
+    MeetupRegistry::<TestRuntime>::iter_prefix_values((
+        cid,
+        EncointerScheduler::current_ceremony_index(),
+    )).collect()
+}
+
+fn get_meetups_by_size(cid: CommunityIdentifier, size: usize) -> Vec<Vec<AccountId>> {
+    get_meetups(cid).into_iter().filter(|meetup| meetup.len() == size).collect()
+}
+
```

Comment:
Just as a general input, collect is relatively expensive, so it might make sense to return iterators here and let the caller decide if we want to collect.

Reply:
Haven't thought of this, but I agree..